apps/api-journeys-modern/src/scripts/test-journey-generation.ts

```
@@ -0,0 +1,36 @@
+import { JourneyGenerationInput } from '../schema/ai/journey-generation.types'
+import { AiJourneyGenerator } from '../models/ai-journey-generator/ai-journey-generator'
```

ESLint

[error] `../models/ai-journey-generator/ai-journey-generator` import should occur before import of `../schema/ai/journey-generation.types`

(import/order)